components/match2/commons/opponent_display.lua

```diff
@@ -306,6 +301,74 @@ function OpponentDisplay.BlockTeam(props)
 		:node(nameNode)
 end
 
+OpponentDisplay.propTypes.InlineParty = {
+	flip = 'boolean?',
+	opponent = Opponent.types.Opponent,
+	overflow = TypeUtil.optional(DisplayUtil.types.OverflowModes),
+	showFlag = 'boolean?',
+	showLink = 'boolean?',
+}
+
+--[[
+Displays a party opponent (solo, duo, trio, or quad) as an inline element.
+]]
+function OpponentDisplay.InlineParty(props)
+	local opponent = props.opponent
```

Comment:
Seems useful to check for null here

Reply:
`props.opponent` is marked required in the type declaration above, so it should be clear to callers that they shouldn't pass nil. Has nil opponents been a problem historically, that we need a dedicated nil check?

I can add a nil check here if needed. Note that without a nil check, a nil opponent would throw on the next line (when accessing `opponent.players`) with a decent error message.